libs/shared/package.json

```diff
@@ -86,6 +86,7 @@
   },
   "dependencies": {
     "@cc-components/sharedcomponents": "workspace:^",
+    "@cc-components/modelviewer": "workspace:^",
```

Comment:
Not saying its wrong but should shared be dependent on modelviewer?

Reply:
Removed!

Reply:
With the current setup, this should remain, because libs/shared contains a shared component for the ModelViewer tab